actors/datacap/src/lib.rs

```diff
@@ -70,7 +70,7 @@ pub enum Method {
     DestroyExported = frc42_dispatch::method_hash!("Destroy"),
     NameExported = frc42_dispatch::method_hash!("Name"),
     SymbolExported = frc42_dispatch::method_hash!("Symbol"),
-    GranularityExported = frc42_dispatch::method_hash!("GranularityExported"),
+    GranularityExported = frc42_dispatch::method_hash!("Granularity"),
```

Comment:
Uh, that's going to be a breaking change for Hyperspace. Hopefully nobody is relying on this method. We will note it in the release notes // cc @jennijuju @maciejwitowski We will need to propagate this change to the Solidity libraries too.